CHANGELOG.md

```diff
@@ -44,7 +45,6 @@
 - Fix that `RATIFY` wasn't total
 - Properly implement treasury withdrawals
 - Remove unused hot credentials at the epoch boundary
-- Implement proper vote counting for SPOs
```

Comment:
@WhatisRT I'm not sure what is the purpose of the WIP section of the CHANGELOG. I moved the issue this PR addresses from the 0.9 section to the WIP section, but I'm not sure if that's what I was meant to do.